meta/mm_sequence.py

```diff
@@ -117,8 +120,13 @@ def update_rqmt(rqmt, extra):
 
         for a_idx, action in enumerate(action_sequence):
             if action.startswith("align"):
+                if use_corrected_applicator:
+                    align_crp, align_extra_config = get_align_config_and_crp_for_corrected_applicator(crp)
+                    align_extra_args["extra_config"] = align_extra_config
```

Comment:
make sure any user decision here does not get lost, so that the user config has the priority